openpype/plugins/publish/help/validate_containers.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+<error id="main">
+<title>Not up-to-date assets</title>
+<description>
+## Obsolete containers found
+
+Scene contains one or more obsolete loaded containers, eg. items loaded into scene by Loader.
+
+### How to repair?
+
+Use 'Scene Inventory' and update all highlighted old container to latest OR
+    refresh Publish and switch 'Validate Containers' toggle on 'Options' tab.
+
+    WARNING: Skipping this validator will result in publishing (and probably rendering) old version of loaded assets.
+</description>
+<detail>
+### __Detailed Info__ (optional)
+
+This validator protects you from rendering obsolete content, someone modified some referenced asset in this scene, eg.
+    by skipping this you would ignore changes to that asset.
```

Comment:
> someone modified some referenced asset in this scene, eg.
>    by skipping this you would ignore changes to that asset.

I don't understand that part of the detailed info. It sounds like "modifications to the referenced asset IN THIS SCENE." but instead it should be about a newer version of the subset being available.

So basically:

- This validates whether you're working with the latest versions of published content loaded into your scene. This protects you from using outdated versions of an asset.

Or something along those lines.